packages/shopify-codemod/test/fixtures/constructor-literal-assignment-to-class-property/complex-literals.output.js

```diff
@@ -0,0 +1,11 @@
+class Foo {
+  foo = {bar: 12, baz: true};
+  bar = {[baz]: 'something'};
```

Comment:
The ESLint rule actually doesn't fail on these, because I don't think ESLint can do the same scope stuff that we can do with jscodeshift. The ESLint rule should only fail for properties that are either string/ boolean/ numeric/ regexp literals, objects/ arrays that are completely simple literals, or method calls on one of those first two.